.github/workflows/test.yml

```diff
@@ -21,7 +21,9 @@ jobs:
         pip install -U pip
 
         curl -sSL \
-          "https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py" | python
+          "https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py" | python -
+
+        echo "${HOME}/.poetry/bin" >> $GITHUB_PATH
```

Comment:
Awesome! Let's also detele all `source "$HOME/.poetry/env"`
They are not needed after this change.

Reply:
Its there now thanks!